var/spack/repos/builtin/packages/tinyxml/package.py

```diff
@@ -32,7 +32,11 @@ class Tinyxml(CMakePackage):
     homepage = "http://grinninglizard.com/tinyxml/"
     url = "https://sourceforge.net/projects/tinyxml/files/tinyxml/2.6.2/tinyxml_2_6_2.tar.gz"
 
-    version('2.6.2', 'cba3f50dd657cb1434674a03b21394df9913d764')
+    version('2_6_2', 'cba3f50dd657cb1434674a03b21394df9913d764')
```

Comment:
sourceforge https://sourceforge.net/projects/tinyxml/files/tinyxml/ lists folders as `2.6.2`, so i would use this.